components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -363,7 +386,7 @@ const ExpenseFormPayeeStep = ({
             </Fragment>
           )}
         </Box>
-        <Box flexGrow="1" flexBasis="50%" display={values.payee?.payoutMethods ? 'block' : 'none'}>
+        <Box flexGrow="1" flexBasis="50%" display={allPayoutMethods ? 'block' : 'none'}>
```

Comment:
This one can create a regression: pick "Invite" then switch expense type. `payee` is now null, and we display the payout form (which should not happen because the fields depend on payee). `allPayoutMethods` is never nullish as it defaults to an empty array.

![image](https://user-images.githubusercontent.com/1556356/143229341-c2f6a419-320b-49cd-95c2-ac972664f97f.png)

Reply:
Would that be ok to look at `allPayoutMethods` length?

I think looking at `allPayoutMethods` make more sense than `values.payee?.payoutMethods`

Reply:
I don't think that would work, because the meanings are different:
- A nullish `values.payee?.payoutMethods` means "no payout profile available" (`payoutMethods` will be an empty array for new users, thus truthy)
- An empty `allPayoutMethods` doesn't make this distinction. If we hide the field when `allPayoutMethods` is empty, it will be hidden for new users that don't have a payout method yet.

Using `Boolean(values.payee)` could work, but we would need to check the impact on expense invites.

Reply:
Will try this

Reply:
I tested and no impact on invites.